db/migrate/20231222161458_change_null_default_created_by_from_participations.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeNullDefaultCreatedByFromParticipations < ActiveRecord::Migration[7.0]
+  def up
+    change_column_null :participations, :created_by, true
+  end
+
+  def down
+    change_column_null :participations, :created_by, false
+  end
+end
```

Comment:
Je ne suis pas sûr de comprendre pourquoi on fait ça. Pour moi on a désormais uniquement besoin de cette colonne comme données historique pour faire nos migrations, donc plutôt que la passer en NOT NULL, on devrait la renommer en `old_created_by`. Très chaud d'en discuter !

Reply:
L'idée était de supprimer la colonne dans une pr suivante (à cause de strong migration) et pour éviter d'avoir des records invalides on est obligé de laisser la possibilité à PG de sauvegarder les records avec un created_by historique nul en attendant de supprimer l'ancienne colonne.

Reply:
Ah ben oui, en effet !